core/panic.c

```diff
@@ -75,7 +76,10 @@ NORETURN void core_panic(core_panic_t crash_code, const char *message)
     panic_arch();
 #ifndef DEVELHELP
     /* DEVELHELP not set => reboot system */
-    reboot();
+    pm_reboot();
+#else
+    /* DEVELHELP set => power off system */
+    pm_off();
```

Comment:
can wen maybe get around the `weak` thing by handling this equal to the idle thread: in case the PM feature is provided we use it, if not simply go into a `while (1) {}` loop?! -> as PM present is the usual case anyway?